examples/rstudio/run.py

```diff
@@ -0,0 +1,4 @@
+import subprocess
+# Running rstudio as a background process with & causes BATS to hang
+# Forking it as a subprocess instead avoids this issue
+subprocess.Popen(['/rstudio/start_rstudio', '8991'])
```

Comment:
I can try to figure out what is going on here. I'd prefer not to have a Python indirection.

Reply:
That'd be great thanks!